svg-injector.js

```diff
@@ -180,14 +172,32 @@
     }
   };
 
+  // Returns SVGSVGElement or undefined
+  var parseSvg = function(svgString){
+    if (DOMParser && (DOMParser instanceof Function)) {
```

Comment:
this is redundant, as the same condition remains in the place where this function is called